### PR TITLE
Co consensus picking

### PR DESCRIPTION
**Pull request by cossorzano**

### CODE REVIEW

File: xmipp3/protocols/protocol_particle_pick_consensus.py [hunk 1]
```diff
@@ -343,7 +345,11 @@ def consensusWorker(coords, consensus, consensusRadius, posFn, jaccFn):
         consensus = Ninputs
     else:
         consensus = consensus.get()
-    consensusCoords = allCoords[votes >= consensus, :]
+    if mode==0:
```
Comment (DStrelak) [inline xmipp3/protocols/protocol_particle_pick_consensus.py]: can this be done differently (enum, maybe)? This approach is error-prone

File: xmipp3/__init__.py [hunk 2]
```diff
@@ -58,7 +58,7 @@ def getEnviron(cls, xmippFirst=True):
         environ.update({
             'PATH': getXmippPath('bin'),
             'LD_LIBRARY_PATH': getXmippPath('lib'),
-            'PYTHONPATH': getXmippPath('pylib')
+            'PYTHONPATH': getXmippPath('pylib')+":"+getXmippPath(os.path.join('pylib','xmippPyModules'))
```
Comment (dmaluenda) [inline xmipp3/__init__.py]: I don't agree on this. Probably the module must be load by
```
from xmippPyModules.moduleName import blablabla
```
or similar.

File: xmipp3/protocols/protocol_particle_pick_consensus.py (hunk 1 above)
Comment (cossorzano) [reply xmipp3/protocols/protocol_particle_pick_consensus.py]: I agree, I have added some labels in the same way as in the rest of protocols

File: xmipp3/__init__.py (hunk 2 above)
Comment (cossorzano) [reply xmipp3/__init__.py]: I cannot remember in which context this error appeared, but this line was the only way we found to solve it.

**Pull request merged**